src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
```

Comment:
Would splitting the imports between the io and util blocks make it more readable?

Reply:
I followed the checkstyle to split between static imports, java imports, org imports and then internal package imports